templates/signup.pug

```diff
@@ -6,8 +6,6 @@ block content
     | Create an account or
     = ' '
     a(href='/sessions/new') login
-    
-  p= error
```

Comment:
Nope actually I now reference it in the layout template! Works perfectly, I just call for "error" on the template.